package.json

```diff
@@ -1,21 +1,59 @@
 {
+  "name": "@lets-talk/root",
   "devDependencies": {
     "@types/jest": "^24.0.0",
     "jest": "^23.4.2",
     "lerna": "^3.0.0-rc.0",
     "microbundle": "^0.12.3",
     "ts-jest": "^24.0.0",
     "tslint": "^5.11.0",
-    "typescript": "^3.0.1"
+    "typescript": "^3.0.1",
+    "commitizen": "4.2.3",
+    "cz-conventional-changelog": "^3.3.0",
+    "husky": "^4.3.8",
+    "lint-staged": "^10.5.3"
   },
   "private": true,
   "scripts": {
-    "bootstrap": "lerna bootstrap",
+    "lerna:build": "lerna run build",
+    "lerna:build:current": "npm run lerna:build --since origin/master --include-filtered-dependents --include-filtered-dependencies",
```

Comment:
What does the --include-filtered-dependents --include-filtered-dependencies does? How it helps us?